imblearn/under_sampling/_prototype_selection/_condensed_nearest_neighbour.py

```diff
@@ -110,17 +110,19 @@ class CondensedNearestNeighbour(BaseCleaningSampler):
     Examples
     --------
     >>> from collections import Counter  # doctest: +SKIP
-    >>> from sklearn.datasets import fetch_mldata  # doctest: +SKIP
+    >>> from sklearn.datasets import make_classification  # doctest: +SKIP
```

Comment:
Instead I would prefer that we use `fetch_openml` and still fetch the pima dataset.
It should look like:

```python
from sklearn.datasets import fetch_openml
from sklearn.preprocesing import scale

X, y = fetch_openml("diabetes", version=1, return_X_y=True)
X = scale(X)
```
In this manner, we should still deal with the same dataset.

Reply:
Used this dataset now, as opposed to
`Resampled dataset shape Counter({-1: 268, 1: 227})  `
in the original example, the counts have now changed to
`Resampled dataset shape Counter({{0: 268, 1: 181}})`.

It also seems that omitting the scaling does not retrieve the counts in the original example. Might be due to a difference to the original, now not anymore findable dataset.

Reply:
Not a huge deal (we might not apply exactly the same scaling). But this is closer to the original example. Updating the labels and counts would be enough here.